lib/hammer_cli_foreman/ping.rb

```diff
@@ -12,6 +12,12 @@ class ForemanCommand < HammerCLIForeman::Command
             field :active, _('Status')
             field :duration_ms, _('Server Response')
           end
+          field :cache, _('cache'), Fields::Label do
+            field :servers, _('servers'), Fields::Collection do
+              field :status, _('Status')
+              field :duration_ms, _('Server Response')
```

Comment:
Should this be "server response time", or should we add some translation like we have in 
https://github.com/theforeman/hammer-cli-foreman/blob/2c191555979732a4dcd395337a11c662943b603a/lib/hammer_cli_foreman/command_extensions/ping.rb#L9